app/src/main/java/com/rockthevote/grommet/ui/eventFlow/EventCanvasserInfo.java

```diff
@@ -109,7 +111,11 @@ private void observeData() {
                     if (effect instanceof CanvasserInfoState.Success) {
                         listener.setState(DETAILS_ENTERED, true);
                     } else if (effect instanceof CanvasserInfoState.Error) {
-                        //todo anything?
+                        Toast.makeText(
+                                getContext(),
+                                R.string.error_updating_canvasser_info,
+                                Toast.LENGTH_LONG
+                        ).show();
```

Comment:
I figured we might as well show anything here. IDK what else we can do unless we implemented some kind of generic error state.